src/firefly/java/edu/caltech/ipac/firefly/server/util/QueryUtil.java

```diff
@@ -534,10 +534,10 @@ public static DataAccessException createEndUserException(String msg, String deta
     }
 
 
-    private static final int DECI_DEF_MAX_POINTS = AppProperties.getIntProperty("decimation.def.max.points", 100000);
+    private static final int DECI_DEF_MAX_POINTS = AppProperties.getIntProperty("decimation.def.max.points", 10000);
```

Comment:
As I see the number is modified, would that be better to use a variable to represent the number instead of hardcode?